views/partials/trial-search-result.html

```diff
@@ -30,7 +38,9 @@ <h1 class="title"><a href="/trials/{{ trial.id }}">{{ trial.public_title }}</a><
     <dd>
       <ul>
         {% for condition in trial.conditions %}
-        <li>{{ condition.name }}</li>
+          <li>{{ condition.name }}</li>
+        {% else %}
+          <p>{{ unknownValue | underscoresToCapitalized }}</p>
```

Comment:
I think HTML like `<ul><p></p></ul>` is invalid, so we either need to check if `trial.conditions` exist and either create the `<ul>` or not (no need for a `<p>` here, as we're already inside the `<dd>`), or add the unknown value inside a `<li>`. I prefer the former.